site/src/components/breadcrumbs.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Breadcrumbs
+description: The Breadcrumb component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FNWYugObOGcAOlekKyjkEkT%2F%25E2%25AD%2590%25EF%25B8%258F-Atlas---Page-Templating%3Fnode-id%3D204%253A1220
+---
+
+# Breadcrumbs
+
+Use breadcrumbs to show navigation from the current location to less specific locations. Note that the final item is not presented.
```

Comment:
Can you clarify the second sentence? In this example, the final item is "Species". Does it mean that I should not see "Species" in the breadcrumb?

Reply:
I mean the current page breadcrumb is not presented. I'll update the wording

Reply:
Updated